src/collection-browser.ts

```diff
@@ -1612,10 +1612,12 @@ export class CollectionBrowser
       this.leftColumn?.style?.removeProperty('height');
     } else {
       const clientTop = this.leftColumn?.getBoundingClientRect().top;
-      this.leftColumn?.style?.setProperty(
-        'height',
-        `${window.innerHeight - (clientTop ?? 0) - 3}px`
-      );
+      const heightValue =
+        clientTop === 0
+          ? null
+          : `${window.innerHeight - (clientTop ?? 0) - 20}px`;
```

Comment:
SUGGESTION: Could comment on what this 20px is.

Reply:
I believe this is equivalent to the top padding of the facet container -- I think it would work to have this in a CSS var instead. Thanks!